Software/tft/README.md

```diff
@@ -0,0 +1,17 @@
+# The Firmware Tool
+This is a simple tool to add header to raw firmware binary. The header is used by driver to check if hardware is compatible with the image. The header is only 16 bytes long and contains PCI Vendor ID, PCI Device ID and PCI Hardware Revision ID values to check. It also contains the CRC value of the raw image.
+
+## Usage
+The tool has several options:
+* `-input <filename>` - Mandatory option, provides filename of raw binary.
+* `-output <filename>` - Mandatory option, provides file name to write a new firmware file with header. If file already exists it will be overwritten.
+* `-vendor <int>` - Mandatory option, provides PCI Vendor ID to add to header.
+* `-device <int>` - Mandatory option, provides PCI Device ID to add to header.
+* `-hw <int>` - Optional, used to provide PCI Hardware Rev ID. Default is 0.
+* `-apply` - Optional. This is used to actually create new (or overwrite) output file with the header in the beginning.
```

Comment:
* `-input <filename>` - Mandatory option, provides a file name of a raw firmware binary.
* `-output <filename>` - Mandatory option, provides a file name to write a new firmware file with a header. If file already exists it will be overwritten.
* `-vendor <int>` - Mandatory option, provides a PCI Vendor ID used in a header.
* `-device <int>` - Mandatory option, provides a PCI Device ID to used in a header.
* `-hw <int>` - Optional, used to provide a PCI Hardware Rev ID. Default is 0.
* `-apply` - Optional. Actually create a new (or overwrite) output file with a header in the beginning.